test/MileageTest.cpp

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file MileageTest.cpp
+ * @brief 走行距離の測定用クラスのテスト
+ * @author TanakaShinnosuke
+ */
+
+#include "Mileage.h"
+#include <math.h>
+#include <gtest/gtest.h>
+
+namespace etrobocon2022_test {
+  TEST(MileageTest, calculateMileage)
+  {
+    double radius = 50;
+    double rightAngle = 30;
+    double leftAngle = 40;
+    double rightWheel = 2 * rightAngle * radius * M_PI / 360;
```

Comment:
```suggestion
    double rightWheelMilage = 2 * rightAngle * radius * M_PI / 360;
```
かな？気に入らなければ無視してもいいです。